src/main/java/org/dcsa/ebl/service/ActiveReeferSettingsService.java

```diff
@@ -1,10 +1,5 @@
 package org.dcsa.ebl.service;
 
-import org.dcsa.core.service.ExtendedBaseService;
-import org.dcsa.ebl.model.ActiveReeferSettings;
-
-import java.util.UUID;
-
-public interface ActiveReeferSettingsService extends ExtendedBaseService<ActiveReeferSettings, UUID> {
```

Comment:
is this used somewhere?

Reply:
Nope, removed it. Only pointed to repository anyway.